app/views/challenges/_form.html.slim

```diff
@@ -0,0 +1,16 @@
+div class="form container"
+	= form_for @challenge do |f|
+		div class="form-group"
+			= f.label :title, "Title" 
+			br
+			= f.text_field :title 
+		div class="form-group" 
+			= f.label :description, "Description"
+			br 
+			= f.text_area :description
+		div class="form-group"	
+			= f.label :score, "Score (1-10, integer only)"
+			br
+			= f.text_field :score
+
+		= f.submit
```

Comment:
Good job with the form and bootstrap classes! Just a number of issues, maybe:
- You may wish to incorporate [simple_form](https://github.com/plataformatec/simple_form) (One big advantage is it removes the need to separately declare labels. Also, it guesses the field type we desire based on the attribute's datatype, so there's no need to specify the field, e.g. `text_field` vs `text_area`)
- In slim, `div` does not need to be explicitly stated. Also, classes are denoted with `.`. For example, you can rewrite line 1 with just `.form.container`
- Indentation seems off again 😅